releases/unreleased/validate-content-before-saving-with-the-changelog-tool.yml

```diff
@@ -0,0 +1,12 @@
+---
+title: Validate content before saving with the changelog tool
+category: added
+author: Venu Vardhan Reddy Tekula <venu@bitergia.com>
+issue: 17
+notes: >
+    This code validates the format of the yml file and gives info
+    about the errors, if any. It gives an option to edit the file
+    incase it if has any mistakes.
+    
+    This feature is necessary to avoid any mistakes by the user
+    while editing the yml file using the changelog tool.
```

Comment:
Instead of writing this as the comment on a commit, try to write it as if the reader were a user that doesn't know anything about the internals of the code. The user should understand what the new functionality is.

Reply:
Okay